app/src/main/java/com/andrew/tasky/presentation/agenda/AgendaFragment.kt

```diff
@@ -117,7 +117,13 @@ class AgendaFragment : Fragment(R.layout.fragment_agenda) {
                             true
                         }
                         R.id.task -> {
-                            // Todo()
+                            navController.navigate(
+                                AgendaFragmentDirections
+                                    .actionAgendaFragmentToTaskDetailFragment(
+                                        null,
+                                        true
+                                    )
```

Comment:
Without named parameters, I have no idea what these values stand for, always use them if you can 😁